aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -181,17 +181,24 @@ To resolve the problem, use ***either*** of the following approaches:
 
 ## Use session affinity (sticky sessions) for server-side webfarm hosting
 
-A Blazor app prerenders in response to the first client request, which creates UI state on the server. When the client attempts to create a SignalR connection, **the client must reconnect to the same server**. When more than one backend server is in use, the app should implement session affinity, also called *sticky sessions*, for SignalR connections.
+A Blazor app prerenders in response to the first client request, which creates UI state on the server. When the client attempts to create a SignalR connection, the client must reconnect to the same server. When more than one backend server is in use, the app should implement session affinity, also called *sticky sessions*, for SignalR connections.
 
 The following error is thrown by an app that hasn't enabled session affinity in a webfarm:
 
-> :::no-loc text="blazor.server.js:1 Uncaught (in promise) Error: Invocation canceled due to the underlying connection being closed.":::
+> :::no-loc text="Uncaught (in promise) Error: Invocation canceled due to the underlying connection being closed.":::
 
-## Server-side Azure SignalR Service
+For more information on session affinity with Azure App Service hosting, see <xref:blazor/host-and-deploy/server#azure-app-service>.
 
-We recommend using the [Azure SignalR Service](xref:signalr/scale#azure-signalr-service) for server-side development hosted in Microsoft Azure. The service works in conjunction with the app's Blazor Hub for scaling up a server-side app to a large number of concurrent SignalR connections. In addition, the SignalR Service's global reach and high-performance data centers significantly aid in reducing latency due to geography.
+## Azure SignalR Service
 
-[Session affinity](#use-session-affinity-sticky-sessions-for-server-side-webfarm-hosting) is enabled for the Azure SignalR Service by setting the service's `ServerStickyMode` option or configuration value to `Required`. For more information, see <xref:blazor/host-and-deploy/server#azure-signalr-service>.
+The optional [Azure SignalR Service](xref:signalr/scale#azure-signalr-service) works in conjunction with the app's SignalR hub for scaling up a server-side app to a large number of concurrent connections. In addition, the service's global reach and high-performance data centers significantly aid in reducing latency due to geography.
+
+The service isn't required for Blazor apps hosted in Azure App Service but can be helpful in other hosting environments:
+
+* To facilitate connection scale out.
+* Handle global distribution.
+
+For more information, see <xref:blazor/host-and-deploy/server#azure-signalr-service>.
```

Comment:
I'd like to generalize this guidance beyond just the Azure App Service case. Basically, you don't get any benefit out of using the Azure SignalR Service with your Blazor Server app if your environment already can handle the WebSocket connection load and is globally distributed. Azure App Service already handles these things, but so does Azure Container Apps and potentially other environments.

```suggestion
The service isn't required for Blazor apps hosted in Azure App Service or Azure Container Apps but can be helpful in other hosting environments:

* To facilitate connection scale out.
* Handle global distribution.

For more information, see <xref:blazor/host-and-deploy/server#azure-signalr-service>.
```